### PR TITLE
Add documentation for new connectors

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

File: en/docs/reusable-components/choreo-connectors-list.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+| **POP3**       | Retrieves emails from a mail server over POP3.                |
+| **IMAP**       | Retrieves emails from a mail server over a TCP/IP connection. |
+
+The following lists the connections you can use to interact with application-specific APIs:
+
+| **Connection**      | **Description**                                                                                |
+|---------------------|------------------------------------------------------------------------------------------------|
+| **Mail by Choreo**  | Send email via Choreo to a specific address with a specific subject and a message body.        |
+| **SMS by Choreo**   | Send SMS via Choreo to a specific telephone number with a specific message body.               |
+| **COVID-19 API**    | Integrate with the COVID-19 API to retrieve statistics relating to the spread of COVID-19 disease. You can perform operations such as retrieving the number of new cases in a specific country, retrieve details relating to the vaccine coverage in a continent, etc. |
```
Comment (anupama-pathirage) [inline en/docs/reusable-components/choreo-connectors-list.md:16]: Why yse `retrieving` and `retrieve` both?

Comment (anupama-pathirage) [reply en/docs/reusable-components/choreo-connectors-list.md:16]: Btw, Can we reduce the length may be: 

Integrate with the COVID-19 API to retrieve the statistics related to the spread of  the disease, such as number of new cases in a specific country, vaccine coverage data, etc.

Comment (RukshiW) [reply en/docs/reusable-components/choreo-connectors-list.md]: Added the suggested test as the description.

File: en/docs/reusable-components/choreo-connectors-list.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+| **POP3**       | Retrieves emails from a mail server over POP3.                |
+| **IMAP**       | Retrieves emails from a mail server over a TCP/IP connection. |
+
+The following lists the connections you can use to interact with application-specific APIs:
+
+| **Connection**      | **Description**                                                                                |
+|---------------------|------------------------------------------------------------------------------------------------|
+| **Mail by Choreo**  | Send email via Choreo to a specific address with a specific subject and a message body.        |
+| **SMS by Choreo**   | Send SMS via Choreo to a specific telephone number with a specific message body.               |
+| **COVID-19 API**    | Integrate with the COVID-19 API to retrieve the statistics related to the spread of the disease, such as number of new cases in a specific country, vaccine coverage data, etc. |
```
Comment (anupama-pathirage) [inline en/docs/reusable-components/choreo-connectors-list.md:16]:
```suggestion
| **COVID-19 API**    | Integrate with the COVID-19 API to retrieve the statistics related to the spread of the disease such as number of new cases in a specific country, vaccine coverage data, etc. |
```

File: en/docs/reusable-components/choreo-connectors-list.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+| **World Bank API**  | Integrate with the World Bank API to retrieve information relating to population, literacy rate, Gross Domestic Product (GDP), government expenditure, etc., in different countries. |
```
Comment (anupama-pathirage) [inline en/docs/reusable-components/choreo-connectors-list.md:17]: `relating to` or `related to`? In COVID 19 we have used `related to`

Comment (RukshiW) [reply en/docs/reusable-components/choreo-connectors-list.md]: Changed the sentence using "relating to". Now it is "related to" in all places.

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**